controller/lib/hal/hal.h

```diff
@@ -402,15 +406,21 @@ inline void HalApi::watchdog_init() {
 #endif
 }
 inline void HalApi::watchdog_handler() {
+  uint32_t millis = ::millis();
```

Comment:
Explain why we do this in the watchdog handler?

Reply:
Done.